ravem/indico_ravem/controllers.py

```diff
@@ -25,58 +30,84 @@ def _check_access(self):
         if not has_access(self.event_vc_room):
             raise RavemException(_("Not authorized to access the room with RAVEM"))
 
-    def _process_args(self):
-        id_ = request.view_args['event_vc_room_id']
-        self.event_vc_room = VCRoomEventAssociation.find_one(id=id_)
-        if not self.event_vc_room:
-            raise NotFound(_("Event VC Room not found for id {id}").format(id=id_))
+    def get_event_vc_room(self, _id):
+        event_vc_room = VCRoomEventAssociation.find_one(id=_id)
+        if not event_vc_room:
+            raise NotFound(_("Event VC Room not found for id {id}").format(id=_id))
+        if not event_vc_room.link_object:
+            raise IndicoError(
+                _("Event VC Room ({id}) is not linked to anything").format(id=_id)
+            )
+        return event_vc_room
 
-        if not self.event_vc_room.link_object:
-            raise IndicoError(_("Event VC Room ({id}) is not linked to anything").format(id=id_))
+    def _process_args(self):
+        _id = request.view_args['event_vc_room_id']
+        self.event_vc_room = self.get_event_vc_room(_id)
 
         event_id = int(request.view_args['confId'])
         event = self.event_vc_room.link_object.event
         if not event:
-            raise IndicoError(_("Event VC Room ({id}) does not have an event").format(id=id_))
-
+            raise IndicoError(
+                _("Event VC Room ({id}) does not have an event").format(id=_id)
+            )
         if event.id != event_id:
-            raise IndicoError(_("Event VC Room ({id}) does not have an event with the id {conf.id}")
-                              .format(id=id_, conf=event))
-
-        room = self.event_vc_room.link_object.room if self.event_vc_room.link_object else None
-        if not room:
-            raise IndicoError(_("Event VC Room ({id}) is not linked to an event with a room").format(id=id_))
-
-        self.room_name = room.generate_name()
-        self.room_special_name = room.name
-
-        if not self.room_name:
-            raise IndicoError(_("Event VC Room ({id}) is not linked to an event with a valid room").format(id=id_))
-
-        if not self.room_special_name:
-            self.room_special_name = self.room_name
+            raise IndicoError(
+                _(
+                    "Event VC Room ({id}) does not have an event with the id {conf.id}"
+                ).format(id=_id, conf=event)
+            )
```

Comment:
Can you keep those formatting-only changes out for now since they'll just make merge conflicts when merging into master more likely unless you already made actual changes in the same code?

Reply:
Yes, sure, I regret having this formatted.